static/styles/left-sidebar.scss

```diff
@@ -316,31 +317,35 @@ li.top_left_starred_messages {
 }
 
 /*
-    The All Messages and Stream chevrons are
+    The All Messages and Stream ellipsis-v(vertical 3 dots) are
     pretty similar.
 */
 .all-messages-sidebar-menu-icon,
 .starred-messages-sidebar-menu-icon,
 .stream-sidebar-menu-icon {
-    top: 3px;
-    right: 10px;
-    font-size: 0.8em;
+    top: 0px;
+    right: 0px;
```

Comment:
Can we just remove the `top` / `right` CSS rather than setting then to 0px?

Reply:
No, Styling will break if I remove them
![Screenshot from 2020-05-01 05-08-09](https://user-images.githubusercontent.com/32801674/80769122-d9cae080-8b69-11ea-911c-d716086ca910.png)